Isosurfacing_3/include/CGAL/Cartesian_grid_3.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+
+#ifndef CGAL_CARTESIAN_GRID_3_H
+#define CGAL_CARTESIAN_GRID_3_H
+
+#include <CGAL/Bbox_3.h>
+#include <CGAL/Image_3.h>
+#include <CGAL/license/Isosurfacing_3.h>
```

Comment:
I think this must be the very first `#include`

Reply:
I'll fix and push, as your gsoc is over.

Reply:
Thanks, I will still try to get though all the comments and fix them. It might just take longer